### PR TITLE
Linting

### PR DESCRIPTION
**Pull request by frelon**

### CODE REVIEW

**Review by davidcassany — APPROVED** [review]
LGTM

File: pkg/types/v1/logger.go [hunk 1]
```diff
@@ -166,4 +166,4 @@ func convert(args []interface{}) string {
 func (w *logrusWrapper) SetContext(string) {}
 func (w *logrusWrapper) Spinner()          {}
 func (w *logrusWrapper) SpinnerStop()      {}
-func (w *logrusWrapper) Screen(t string)   {}
+func (w *logrusWrapper) Screen(_ string)   {}
```
Comment (fgiudici) [inline pkg/types/v1/logger.go:169]: nit, format inconsistency: few lines above ( line 166), we  use unnamed parameter, here we use the blank identifier.

File: pkg/types/v1/logger.go [hunk 2]
```diff
@@ -112,8 +112,8 @@ func (w *logrusWrapper) Ask() bool {
 
 func (w *logrusWrapper) Copy() (types.Logger, error) {
 	c := *w
-	copy := &c
-	return copy, nil
+	copied := &c
+	return copied, nil
```
Comment (fgiudici) [inline pkg/types/v1/logger.go:116]: 🤔 just curious, was the linter complaining here for the var name?

**Review by fgiudici — APPROVED** [review]
looks good here!

File: pkg/types/v1/logger.go (hunk 2 above)
Comment (frelon) [reply pkg/types/v1/logger.go:116]: Yep, it's shadowing the builtin copy function.

**Pull request merged**